presto-hive/src/main/java/com/facebook/presto/hive/BackgroundHiveSplitLoader.java

```diff
@@ -145,6 +144,7 @@ public BackgroundHiveSplitLoader(
         this.executor = executor;
         this.partitions = new ConcurrentLazyQueue<>(partitions);
         this.hdfsContext = new HdfsContext(session, table.getDatabaseName(), table.getTableName());
+        this.corruptBucketing = "Hive table (" + table + ") is corrupt. It is declared as being bucketed, but the files do not match the bucketing declaration.";
```

Comment:
Please do not construct exception message for some pessimistic case in class constructor. If you want to use same message twice, you can have a method formatting the message.